osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -65,6 +65,7 @@ protected override void InitialiseDefaults()
 
             // Gameplay
             Set(OsuSetting.DimLevel, 0.3, 0, 1, 0.01);
+            Set(OsuSetting.BlurLevel, false);

```

Comment:
Either make this variable similar to `BlurLevel` or rename it to `GameplayBackgroundBlur`.